packages/wrangler/src/d1/migrations/apply.tsx

```diff
@@ -116,7 +138,7 @@ Your database may not be available to serve requests during the migration, conti
 		if (!ok) return;
 
 		// don't backup prod db when applying migrations locally or in preview
-		if (!local && !preview) {
+		if (!local && !preview && !experimental) {
```

Comment:
```suggestion
		if (local || preview || experimental) {
```

Reply:
but we don't want to run a backup, when any of them are true?

Reply:
But only a personal preference here.